README.md

```diff
@@ -6,6 +6,15 @@
 This project adds a REST API around the Replikator that makes it possible to manage the replikated databases using a
 REST api.
 
+```
+Usage of replikator-api:
+	Restfull Replikator API server
+Options:
+  -l, --listen=:8080                    listen address
+  -r, --replikator=sudo replikator-ctl  Path to replikator-ctl
```

Comment:
There should be quotes here right?

Reply:
hmm good one. This is a copy/paste from `--help`